### ISSUE TITLE
Uncaught TypeError: Cannot read property 'path' of null

### ISSUE THREAD

**Issue by Bomret** [fsprojects/FSharp.Atom#45]
Fresh install of atom. The status bar is stuck on "F# Status - Loading". When i create a new F# file and begin to type, the following error occurs.

**Atom Version**: 0.199.0
**System**: WKS-SR
**Thrown From**: [FSharp](https://github.com/fsprojects/FSharp.Atom) package, v0.2.5

### Stack Trace

Uncaught TypeError: Cannot read property 'path' of null

```
At C:\Users\sr.COPRA\.atom\packages\FSharp\lib\fsharp.js:474

TypeError: Cannot read property 'path' of null
    at AutocompleteProvider__getSuggestion$ (C:\Users\sr.COPRA\.atom\packages\FSharp\lib\fsharp.js:474:47)
    at AutocompleteProvider__create$ [as getSuggestions] (C:\Users\sr.COPRA\.atom\packages\FSharp\lib\fsharp.js:241:14)
    at C:\Users\sr.COPRA\AppData\Local\atom\app-0.199.0\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:254:56
    at Array.forEach (native)
    at AutocompleteManager.module.exports.AutocompleteManager.getSuggestionsFromProviders (C:\Users\sr.COPRA\AppData\Local\atom\app-0.199.0\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:233:17)
    at AutocompleteManager.getSuggestionsFromProviders (C:\Users\sr.COPRA\AppData\Local\atom\app-0.199.0\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:3:61)
    at AutocompleteManager.module.exports.AutocompleteManager.findSuggestions (C:\Users\sr.COPRA\AppData\Local\atom\app-0.199.0\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:221:19)
    at C:\Users\sr.COPRA\AppData\Local\atom\app-0.199.0\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:3:61
```

### Commands

```
     -0:19.2.0 grammar-selector:show (atom-text-editor.editor)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
FSharp, v0.2.5
language-batch, v0.3.3
language-fsharp, v0.8.4
Paket, v0.2.2

# Dev
No dev packages
```

**Comment by Krzysztof-Cieslak** [comment]
Error messages should be fixed in `0.2.6`. However to get features such as autocomplete, error highlighting, You need to save file to disc, it can't be Atom in-memory file. Preferably save file and open it again to be sure everything is working good.

Sorry for inconvenience will try to make better user experience with new files in the future.

**Comment by Bomret** [comment]
Ah ok. Thanks, I'll close this ticket.
